documentation/general/dash-sonic-hld.md

```diff
@@ -1213,7 +1257,20 @@ For the inbound direction, after Route/ACL lookup, pipeline shall use the "under
         "OP": "SET"
     },
     {
-        "DASH_ROUTE_TABLE:F4939FEFC47E:50.1.2.0/24": {
+        "DASH_ENI_ROUTE_TABLE:F4939FEFC47E": {
+	    "group_id":"group_id_2"
+        },
```

Comment:
Different ENI ids for group_id_1, group_id_2 and group_id_3.

Reply:
This is a reference example. it can be any group_id